SectionedCollectionView/Views/CollectionViewCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  CollectionViewCell.swift
+//  SectionedCollectionView
+//
+//  Created by Esteban Arrua on 6/8/18.
+//  Copyright © 2018 Hattrick. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CollectionViewCell: UICollectionViewCell {
+    
+    static let nibName = "CollectionViewCell"
+    static let cellReuseIdentifier = "CollectionViewCell"
+    
+    private let gradientStartColor = UIColor(red: 46/255, green: 132/255, blue: 250/255, alpha: 1)
+    private let gradientEndColor = UIColor(red: 83/255, green: 99/255, blue: 236/255, alpha: 1)
+    private var gradientBackgroundLayer: CAGradientLayer?
+    
+    @IBOutlet weak var nameLabel: UILabel!
+    
+    func configure(with name: String, selected: Bool) {
+        nameLabel.text = name
+        nameLabel.textColor = selected ? .white : .black
+        backgroundColor = .white
+        layer.cornerRadius = 20
+        setupGradientBackground(selected: selected)
+    }
+    
+    private func setupGradientBackground(selected: Bool) {
+        let prevGradientBackgroundLayer = gradientBackgroundLayer
```

Comment:
what if you use just one gradientLayer item and hide it instead of removing it? it could be more efficient